File: BRACU ACM Student Chapter/Online Workshop on Programming Skills DAY 01 ( C++ ) BRACU ACM Student chapter/R.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string input;
    cin >> input;
    for(int i=1; i<input.length(); i++)
    {
        if(input[i]==input[i-1])
        {
            input.erase(i-1,2);
            i=0;
        }
    }
    input.length()==0 ? cout << "Empty String" : cout << input;
    return 0;
}

File: BRACU ACM Student Chapter/Online Workshop on Programming Skills DAY 01 ( JAVA ) BRACU ACM Student chapter/R.java
import java.util.Scanner;
public class R {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    String word = input.next();
    System.out.println(delete(word));
  }
  
  static String delete(String word) {
    StringBuilder input = new StringBuilder(word);
    boolean condition = true;
    
    while (condition) {
      condition = false;
      for (int i = 0; i < input.length() - 1; ++i) {        
        if (input.charAt(i) == input.charAt(i + 1)) {
          input.delete(i, i + 2);
          condition = true;
          
        }
      }
    }
    
    if (input.length() == 0) return "Empty String";
    else return input.toString();
  }  
}